src/components/FormInput.tsx

```diff
@@ -5,22 +5,38 @@ interface FormInputProp {
 }
 
 const FormInput: React.FC<FormInputProp> = ({ addTask }) => {
-
     const [ userInput, setUserInput ] = useState('');
-
+    const [ inputError, setInputError ] = useState('');
+    const errorCheck = () => {
+        if (!userInput) {
+            setInputError("No blank spaces");
+            return false;
+        }
+        // if (userInput === fullList) {
+        //     setInputError("Item already on list.");
+        // }
+        return true;
+    }
     const handleChange = (e: { currentTarget: { value: React.SetStateAction<string>; }; }) => {
         setUserInput(e.currentTarget.value)
     }
 
     const handleSubmit = (e: { preventDefault: () => void; }) => {
         e.preventDefault();
-        addTask(userInput);
-        setUserInput("");
-    }
-    
+        const isValid = errorCheck();
+        if(!isValid) {
+            setUserInput("");
+        }
+        else{
+            setInputError('');
```

Comment:
could take this line out of the if else loop and place outside

Reply:
Moved out of if/else